VRCFaceTracking/ConfigParser.cs

```diff
@@ -62,10 +62,11 @@ public static void ParseNewAvatar(string newId)
             Logger.Msg("Parsing config file for avatar: " + avatarConfig.name);
             var parameters = avatarConfig.parameters.Where(param => param.input != null).ToArray();
 
+            int count = 0;
             foreach (var parameter in UnifiedTracking.AllParameters_v2.Concat(UnifiedTracking.AllParameters_v1).ToArray())
-                parameter.ResetParam(parameters);
+                count += parameter.ResetParam(parameters);
```

Comment:
Where we could detect loaded AllParameter versions as well for the ConfigLoaded action.